lib/browser/api/browser-actions.js

```diff
@@ -62,6 +64,6 @@ module.exports = {
   },
 
   getPopup: (extensionId, details) => {
-    return getBrowserAction(extensionId, details).popup
+    return getBrowserAction(extensionId, details).popup || getBrowserAction(extensionId, {}).popup
```

Comment:
I don't think this extra call is necessary. I think all you need here is a default value for `details` in `getBrowserAction`